hdr_html_canvas_element.md

```diff
@@ -125,13 +125,33 @@ The component signals are mapped to red, green and blue tristimulus values accor
 * Transfer function:
 
 ```
-   E = | E' / 12.92, if abs(E') ≤ 0.04045
-       | ((E' + 0.055) / 1.055)^2.4, otherwise
-
-       with E' ∈ ℝ
+decoding equation (non-linear to linear):
```

Comment:
I would prefer to only have a single function defined, and to indicate that the conversion from space X to Y is the inverse function of the conversion of space Y to X.  This behavior (that transformations are always inverses) is a bit in contrast to how some things are done in the broadcast world, and I'd like to be very clear about this behavior, and adding the definition of the inverse (at least for now) might take away from that emphasis.

Reply:
Per 2021-07-19: make the decoding equation normative and the encoding equation informative/example.